src/components/Checkbox.scss

```diff
@@ -92,6 +97,12 @@ $Checkbox-bg-color: $cu-foreground !default;
   }
 }
 
+.Checkbox-input:disabled:checked + .Checkbox-label {
+  &:before {
+    background-color: $cu-positive;
+  }
```

Comment:
Just curious - what scenario is this for?

Reply:
if the option is selected and can't be unchecked - it's come up a couple of times